helmfile/charts/networkpolicy-workload/templates/cert-manager/certmanager.yaml

```
@@ -0,0 +1,70 @@
+{{ if .Values.certManager.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager
+  namespace: cert-manager
+spec:
+  podSelector:
+    matchLabels:
+      app: cert-manager
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/instance: kube-prometheus-stack-prometheus
+      ports:
+        - port: 9402
+          protocol: TCP
+  egress:
+    - ports:
+      - port: 53
+        protocol: UDP
+    - to:
+      {{- range $IP := .Values.certManager.letsencrypt.ips }}
+      - ipBlock:
+          cidr: {{ $IP }}
+      {{- end }}
+      ports:
+        - protocol: TCP
+          port: {{ .Values.certManager.letsencrypt.port }}
+    - to:
+        {{- range $IP := .Values.global.wcApiserver.ips }}
+        - ipBlock:
+            cidr: {{ $IP }}
+        {{- end }}
+      ports:
+        - protocol: TCP
+          port: {{ .Values.global.wcApiserver.port }}
+    - to:
+    {{- if .Values.global.externalLoadBalancer }}
+        {{- range $IP := .Values.global.wcIngress.ips }}
+        - ipBlock:
+            cidr: {{ $IP }}
+        {{- end }}
+    {{- else if .Values.global.ingressUsingHostNetwork }}
+        {{- range $IP := .Values.global.wcNodes.ips }}
+        - ipBlock:
+            cidr: {{ $IP }}
+        {{- end }}
+    {{ else }}
+        - namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: ingress-nginx
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/component: controller
+              app.kubernetes.io/instance: ingress-nginx
+  {{- end }}
```

To match the parent if/else
```suggestion
    {{- end }}
```